src/lib/operators/print.hpp

```diff
@@ -8,7 +8,8 @@
 #include "abstract_read_only_operator.hpp"
 
 namespace opossum {
-enum PrintFlags { PrintMvcc = 1 << 1 };
+
+enum PrintFlags { PrintMvcc = 1 << 1, PrintIgnoreChunkBoundaries = 1u << 2u };
```

Comment:
Haven't heard of these chunk boundaries before. It's just about whether we print where a chunk ends and the other begins?

Reply:
precisely. For chunk_size = 1 (which happens in tests) it's hard to decipher table printings without this flag. will doc it.